dicom-mr-classifier.py

```diff
@@ -574,6 +574,16 @@ def dicom_classify(zip_file_path, outbase, timezone, config=None):
     if hasattr(dcm, "Modality") and dcm.get("Modality"):
         metadata["acquisition"]["instrument"] = format_string(dcm.get("Modality"))
 
+    # If no pixel data present, make classification intent "Non-Image"
+    if not hasattr(dcm, "PixelData"):
+        nonimage_intent = {"Intent": ["Non-Image"]}
+        # If classification is a dict, update dict with intent
+        if isinstance(dicom_file["classification"], dict):
+            dicom_file["classification"].update(nonimage_intent)
+        # Else classification is a list, assign dict with intent
+        else:
+            dicom_file["classification"] = nonimage_intent
+
     series_desc = format_string(dcm.get("SeriesDescription", ""))
```

Comment:
Do we need to escape this code if the file is non-img?

Reply:
What do you mean by escape?

Reply:
Should the gear skip line 587-ff if it is non-image?

Reply:
I guess this re-ordering actually has implications for the classification calls line 597 and 599. Those should be "update"? and 582 should instantiate the dictionary?

Reply:
You're right, reorganization incoming